addons/popochiu/engine/objects/gui/components/dialog_text/dialog_text.gd

```diff
@@ -169,46 +173,37 @@ func _modify_size(_msg: String, _target_position: Vector2) -> void:
 
 
 ## Creates a RichTextLabel to calculate the resulting size of this node once the whole text is shown.
+## Uses a RichTextLabel to provide the "get_parsed_text" function, and a label within it to work out
+## the minimum size. Calculating the size from just the RichTextLabel does not work.
 func _calculate_size(msg: String) -> Vector2:
 	var rt := RichTextLabel.new()
 	rt.add_theme_font_override("normal_font", get_theme_font("normal_font"))
 	rt.bbcode_enabled = true
 	rt.autowrap_mode = TextServer.AUTOWRAP_WORD_SMART
 	rt.text = msg
-	rt.size = get_meta(DFLT_SIZE)
 	rich_text_label.add_child(rt)
 	
 	# Create a Label to check if the text exceeds the wrap_width
 	var lbl := Label.new()
 	lbl.add_theme_font_override("normal_font", get_theme_font("normal_font"))
 	
-	_set_default_label_size(lbl)
-	
 	lbl.text = rt.get_parsed_text()
+	lbl.size = lbl.get_minimum_size()
```

Comment:
I think this is not necessary since the `custom_minimum_size` of the component at this point will be **Vector2.ZERO**.